src/SSW.SophieBot/SSWSophieBot/dialogs/GetFreeDayDialog/language-generation/en-us/GetFreeDayDialog.en-us.lg

```diff
@@ -64,7 +62,9 @@
         {
             "type": "ActionSet",
             "actions": [
-                    ${ViewCalendarAction(employee.displayName, employee.bookedDays, 'GetFreeDay')}
+                    ${MoreBookingInfoAction()},
+                    ${ViewCalendarAction(employee.displayName, employee.bookedDays, 'GetFreeDay')},
+                    ${ViewProfileCardAction(employee.firstName, employee.lastName)}

```

Comment:
Please add intentredirect entity in action. See [here](https://github.com/SSWConsulting/SSW.SophieBot/blob/416aca86033b16be28151fbe6327ee8f68b4a5df/src/SSW.SophieBot/SSWSophieBot/language-generation/en-us/common.en-us.lg#L22) as a reference

Reply:
Done. Check this commit: https://github.com/SSWConsulting/SSW.SophieBot/pull/293/commits/f1a5c7932d8cdec56be8a60d7c6ea99d7d283a5c

Reply:
> Please add `intentredirect` entity in action. See [here](https://github.com/SSWConsulting/SSW.SophieBot/blob/416aca86033b16be28151fbe6327ee8f68b4a5df/src/SSW.SophieBot/SSWSophieBot/language-generation/en-us/common.en-us.lg#L22) as a reference

Done. Check this commit: https://github.com/SSWConsulting/SSW.SophieBot/pull/293/commits/f1a5c7932d8cdec56be8a60d7c6ea99d7d283a5c